pkg/virt-controller/watch/migration.go

```diff
@@ -369,6 +370,20 @@ func (c *MigrationController) canMigrateVMI(migration *virtv1.VirtualMachineInst
 
 }
 
+func handleFailedMigrationMetrics(migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) {
+	if phase := &migration.Status.Phase; phase != nil {
```

Comment:
There's no need to define `phase` as a pointer then check whether it's nil.
You can go with:
```go
if phase := migration.Status.Phase {
...
}
```

Reply:
updated